hangman.js

```diff
@@ -1 +1,28 @@
-console.log("welcome to hangman");
\ No newline at end of file
+function playHangman(){
+  console.log("welcome to hangman");
+  words = ['javascript','ruby','rails','jquery','programming'];
+  dashedWords = ['- - - - - - - - - -','- - - -','- - - - -','- - - - - -','- - - - - - - - - - -'];
+  guesses = [];
+  incorrectGuesses = 0;
+function randomizeIndex(){
+    randomIndex = Math.floor(Math.random()*5);
+  }
+randomizeIndex();
+}
+function randomWord(){
+  word = words[randomIndex];
+  dashedWord = dashedWords[randomIndex];
+  dashedArray = dashedWord.split(" ");
+  console.log(dashedArray);
+  }
+function guessLetter(){
```

Comment:
How is this function being utilized? Can you think of a way to call this function until some condition is met?